### PR TITLE
Deprecate GELF

### LINKED ISSUE #27364: Deprecate GELF
### Description

Keycloak uses the [logstash-gelf](https://github.com/mp911de/logstash-gelf) dependency which has reached EOL. As we don't see much community adoption of this feature (based on the bug reports), we should deprecate the GELF log handler.

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by vmuzikar**
Closes #27364

This PR also adds the possibility to deprecate individual values, not just the whole options. Preview of how it looks like:
Screenshot 2024-02-28 at 17 09 34
Screenshot 2024-02-28 at 17 08 17
Screenshot 2024-02-28 at 17 07 43

### CODE REVIEW

**Review by mabartos — APPROVED** [review]
@vmuzikar Nice job! LGTM

**Review by ahus1 — APPROVED** [review]
Approving based on @mabartos's review. Leaving it for the CND team to merge this when they see this fit.

**Comment by mabartos** [comment]
@ahus1 Thanks.

IMO, the PR is really straightforward and does not need to have more reviews from the CDN team.

Merging now.

**Pull request merged**
